packages/android_alarm_manager_plus/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.4
+
+- Fix FlutterEngine initialization
```

Comment:
I think we should be more explicit here, so people could understand that this annoying bug is finally resolved.
```suggestion
- Fix FlutterEngine initialisation to avoid crashes when `rescheduleOnReboot` is set to true
```